src/js/invertedIndex.js

```diff
@@ -106,13 +112,14 @@ class InvertedIndexClass {
 
     try {
       inputFile.forEach((thisBook) => {
-        const hasTitle = Object.hasOwnProperty.call(thisBook, 'title');
-        const hasText = Object.hasOwnProperty.call(thisBook, 'text');
+        const hasTitle = Object.prototype.hasOwnProperty.call(thisBook,
+        'title');
+        const hasText = Object.prototype.hasOwnProperty.call(thisBook, 'text');
         if (!(hasTitle && hasText)) {
           this.inputFile = null;
```

Comment:
why not `this.input = false` so that ` return this.inputFile || false;` can be  `return this.inputFile;`